scipy/interpolate/meson.build

```diff
@@ -151,12 +164,20 @@ py3.extension_module('_fitpack',
   subdir: 'scipy/interpolate'
 )
 
-# TODO: Add flags for 64 bit ints
+if use_ilp64
+   # generator only accepts strings, not files
+   f2c_map_file = f2py_ilp64_opts[1]
+   extra_arg = f2py_ilp64_opts[0] + '=' + fs.parent(f2c_map_file) / fs.name(f2c_map_file)
+  _dfitpackmodule_obj = f2py_gen.process('src/dfitpack.pyf', extra_args: extra_arg)
```

Comment:
This deserves an explanation in a code comment for why only `dfitpack` needs this `f2c_map_file`.